job-service-container/src/integration-test/java/com/hpe/caf/services/job/api/JobServiceIT.java

```diff
@@ -97,6 +111,65 @@ public void setup() throws Exception {
         rabbitConfiguration.getRabbitConfiguration().setRabbitPort(Integer.parseInt(SettingsProvider.defaultProvider.getSetting(SettingNames.rabbitmqNodePort)));
     }
 
+    @Test
+    public void testHealthCheck() throws NoSuchAlgorithmException, KeyManagementException, IOException
+    {
+        final String getRequestUrl = SettingsProvider.defaultProvider.getSetting("healthcheckurl");
+        final HttpGet request = new HttpGet(getRequestUrl);
+
+        // Set up a trust-all cert manager implementation
+        final TrustManager[] trustAllCertsManager = new TrustManager[]{
+                new X509TrustManager()
+                {
+                    @Override
+                    public java.security.cert.X509Certificate[] getAcceptedIssuers()
+                    {
+                        System.out.println("Trust All TrustManager getAcceptedIssuers() called");
+                        return null;
+                    }
+
+                    @Override
+                    public void checkClientTrusted(
+                            java.security.cert.X509Certificate[] certs, String authType)
+                    {
+                        System.out.println("Trust All TrustManager checkClientTrusted() called");
+                    }
+
+                    @Override
+                    public void checkServerTrusted(
+                            java.security.cert.X509Certificate[] certs, String authType)
+                    {
+                        System.out.println("Trust All TrustManager CheckServerTrusted() called");
+                    }
+                }
+        };
+
+        // Instantiate an SSLContext object which employs the trust-all cert manager.
+        final SSLContext sslContext = SSLContext.getInstance("SSL");
+        sslContext.init(null, trustAllCertsManager, new java.security.SecureRandom());
+
+        // Set our HttpClient's Hostname Verifier to the no-op hostname verifier which turns hostname verification off.
+        final HttpClient httpClient = HttpClients.custom().setSSLHostnameVerifier(NoopHostnameVerifier.INSTANCE)
+                .setSSLContext(sslContext).build();
+
+        System.out.println("Sending GET to HealthCheck url: " + getRequestUrl);
+        final HttpResponse response = httpClient.execute(request);
+        request.releaseConnection();
+
+        if (response.getEntity() == null) {
+            Assert.fail("There was no content returned from the HealthCheck HTTP Get Request");
+        }
+
+        final String expectedHealthCheckResponseContent =
+                "{\"database\":{\"healthy\":\"true\"},\"queue\":{\"healthy\":\"true\"}}";
+        Assert.assertEquals("Expected HealthCheck response should match the actual response",
+                expectedHealthCheckResponseContent,
+                IOUtils.toString(response.getEntity().getContent(), Charset.defaultCharset()));
```

Comment:
Don't use `defaultCharset()` as it is platform specific.  Use one of the standard character sets - either `ISO_8859_1` or `UTF_8`.